src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -1,74 +1,33 @@
-import React from 'react';
-import isEqual from 'lodash/isEqual';
+import React, {useMemo} from 'react';
 import {ThemeContext} from '../../utilities/theme';
-import {Theme} from '../../utilities/theme/types';
-import {setColors} from '../../utilities/theme/utils';
-
-interface State {
-  theme: Theme;
-  colors: string[][] | undefined;
-}
+import {ThemeConfig} from '../../utilities/theme/types';
+import {
+  buildThemeContext,
+  buildCustomProperties,
+} from '../../utilities/theme/utils';
```

Comment:
This should export from `'../../utilities/theme'`

Reply:
Loving that these these transformation functions now live in utilities/theme, It makes loads more sense!